lib/dialogs.dart

```diff
@@ -85,26 +80,49 @@ class EmbedDialog extends DDialog {
     _configure();
     super.show();
   }
-  
+
   void generateExport() {
     MutableGist gist = gistContainer.mutableGist;
     Uri url = Uri.parse(window.location.toString());
     String home = url.host;
-    _doc.add(new SpanElement()..text = 'Dart + Documentation: ' ..style.paddingRight = "12px");
-    _doc.add(new InputElement()..value = '${home}/embed-dart.com?id=${gist.id}' ..attributes['flex'] = '');
-    _html.add(new SpanElement()..text = "Dart + Html: " ..style.paddingRight = "12px");
-    _html.add(new InputElement()..value = '${home}/embed-html.com?id=${gist.id}' ..attributes['flex'] = '');
-    _inline.add(new SpanElement()..text = "Dart (Minimal): " ..style.paddingRight = "12px");
-    _inline.add(new InputElement()..value = '${home}/embed-inline.com?id=${gist.id}' ..attributes['flex'] = '');
+    _doc.add(new SpanElement()
+      ..text = 'Dart + Documentation: '
```

Comment:
Can you instead add a class to this, and have the class have the padding and min-width settings? The class would probably need to be defined in web/index.css.